### ISSUE TITLE
contrast limits slider broken with very small max value

### ISSUE THREAD

**Issue by sofroniewn** [napari/napari#3893]
## 🐛 Bug

I was working with an image today with a very small max value, 2.6e-17, but a lot of richness around 10-20 etc, and found that the contrast limits slider is not adjustable, i.e. when i click on it I can't drag it around.

## To Reproduce

```python
v = napari.Viewer()
v.add_image(np.random.random((10, 10))*2.6e-17)
```

## Expected behavior

I should be able to adjust contrast limits value

## Environment

Current main, and latest superqt

**Comment by tlambert03** [comment]
If you can produce this with only superqt, can you open an issue there instead?

**Comment by sofroniewn** [comment]
PR fix created on the napari side #3895
